src/components/FeedActivity.jsx

```diff
@@ -0,0 +1,96 @@
+import { Avatar,  Divider, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+      width: "100%"
+  },
+    avatar: {
+    marginTop: theme.spacing(2),
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(2)
+  },
+  name: {
+    marginBottom: theme.spacing(1),
+    fontSize: 14
+  },
+  stations: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  date: {
+    marginTop: theme.spacing(0),
+    marginLeft: theme.spacing(3),
+    color: theme.palette.grey[600],
+  },
+  vertDivider: {
+    backgroundColor: theme.palette.primary.main,
+    width: "2px",
+    marginLeft: theme.spacing(3),
+    marginTop: theme.spacing(1)
+  },
+  time: {
+    marginTop: theme.spacing(0),
+    marginBottom: theme.spacing(0),
+    color: theme.palette.grey[600],
+  },
+  container: {
+    display: "flex",
+  },
+  divider: {
+      width: "100%",
+      left: theme.spacing(2)
+  },
+  duration: {
+    color: theme.palette.grey[600],
+    marginLeft: theme.spacing(2)
+  },
+  activityExcercices: {
+    display: "flex",
+    flexDirection: "column",
+    marginLeft: theme.spacing(1),
+    marginTop: theme.spacing(1)
+  },
+  number: {
+    marginRight: theme.spacing(2)
+  }
+}));
+
+function FeedActivity() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.container}>
+          <Avatar
+            className={classes.avatar}
+            alt="Anette Stark"
+            src="https://images.sportbuzzer.de/v1/photos/raw/dsc04610/regular-16-9"
+          />
+          <p className={classes.name}><strong>Name</strong> <br/> x Stationen, x Übungen </p>
```

Comment:
```suggestion
          <div className={classes.header}>
            <div className={classes.name}>Name</div>
            <Typography variant="subtitle2">x Stationen, x Übungen</Typography>
          </div>
```
Diese p-Tags machen einige Probleme, weil die von sich aus schon Margins mitbringen^^
Ich würde hier das Header-Div reinmachen das ich angesprochen hatte, damit wir Avatar und Text zentrieren können.
Die p-Tags habe ich durch ein gestylten Div + ein subtitle2 ersetzt. Das passt ganz gut denke ich.